### ISSUE TITLE
Support New Checks API

### ISSUE THREAD

**Issue by tbrand** [danger/danger#984]
# Proposal

## What did you do?

I've read a blog from Github team.

## What did you expect to happen?

There is already an issue about it.

## What happened instead?

No proposals for it. So I've opened this issue.

## Your Environment

danger for ruby.

## Appendix

The blog is here.
https://developer.github.com/changes/2018-05-07-new-checks-api-public-beta/

**Comment by orta** [comment]
This needs to have a GitHub App for authentication, which will make a lot more time consuming on setup, so whoever makes these will need to take that side of documentation into account.

I plan to take a look at supporting this for Peril with Danger JS, but whether people running Danger on a CI run will be able to use it is still an unknown to me until I have built it out.

**Comment by orta** [comment]
Looking at those PRs, I doubt it, this feature only exists inside danger-js. I'd recommend logging in as the account and seeing if it's been denied access to GitHub

**Comment by jhoareau** [comment]
@orta Any update on porting this feature from danger-js to danger?

**Comment by orta** [comment]
Nope, from my perspective Danger ruby is done and any new feature work from me is on danger-js - I'm open to others adding these kind of things but here I won't be the one pushing it forwards.
